readme.md

```diff
@@ -69,17 +73,35 @@ Each style has an `open` and `close` property.
 
 By default you get a map of styles, but the styles are also available as groups. They are non-enumerable so they don't show up unless you access them explicitly. This makes it easier to expose only a subset in a higher-level module.
 
-- `ansi.modifiers`
-- `ansi.colors`
-- `ansi.bgColors`
+- `style.modifiers`
+- `style.color`
+- `style.bgColor`
 
 
 ###### Example
 
 ```js
-console.log(ansi.colors.green.open);
+console.log(style.color.green.open);
 ```
 
+## 256 / 16 million (TrueColor) support
```

Comment:
Maybe link up `256` and `16 million` to some pages where the user can read more about them?